config/io.github.insilmaril.vym.appdata.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+    <id>io.github.insilmaril.vym</id>
+    <metadata_license>CC0-1.0</metadata_license>
+    <project_license>LGPL-2.1-or-later</project_license>
+    <name>vym</name>
+    <summary>Tool for generating and manipulating mind maps</summary>
+    <description>
+	<p>
+	    VYM (View Your Mind) allows you to easily and intuitively
+            organize your ideas in a structured way. You can create 
+            metal maps and export them to multiple formats.
```

Comment:
"mental" maps

Maybe also worth mentioning: "vym is also powerful personal task manager"